packages/lumx-react/src/components/date-picker/DatePickerControlled.tsx

```diff
@@ -84,9 +125,38 @@ export const DatePickerControlled: Comp<DatePickerControlledProps, HTMLDivElemen
                     />
                 }
                 label={
-                    <span className={`${CLASSNAME}__month`} aria-live={labelAriaLive}>
-                        {selectedMonth.toLocaleDateString(locale, { year: 'numeric', month: 'long' })}
-                    </span>
+                    <>
+                        <span aria-live={labelAriaLive} className="visually-hidden" dir="auto">
+                            {monthYear.split(selectedYear).map((part) => (!part ? selectedYear : part))}
```

Comment:
You don't need to replace the year by the selected year. I would even say we should not, it will update on every change of the input number, it would be better if it is only updated on Enter or blur, when the change is effective. 
The year will be read by the input number when changed.

Reply:
`selectedYear` is only updated when `selectedMonth` is updated which means that it reads only when pressing enter or on blur. I have just checked again using NVDA and it works as described above.

Reply:
As discussed on Slack, replace this by `monthYear`... Shame on me for not noticing this....